.circleci/config.yml

```diff
@@ -16,19 +27,55 @@ jobs:
       - checkout
       - run:
           name: Install deps
-          command: npm install
+          command: |
```

Comment:
Nit: It would be nice to standardize on either using `|` or not for single-line commands. it looks like we're switching between both.